uisdk/src/main/java/com/karhoo/uisdk/screen/booking/map/BookingMapView.kt

```diff
@@ -309,11 +310,9 @@ class BookingMapView @JvmOverloads constructor(
         bundle: Bundle?,
         lifecycleOwner: LifecycleOwner,
         journeyDetailsStateViewModel: JourneyDetailsStateViewModel,
-        shouldReverseGeolocate: Boolean = true,
-        isDeepLink: Boolean = false
     ) {
-        this.isDeepLink = isDeepLink
-        this.shouldReverseGeolocate = if (isLocateMeEnabled) shouldReverseGeolocate else false
+        this.isDeepLink = RidePlanningStorage.journeyInfo != null
+        this.shouldReverseGeolocate = if (isLocateMeEnabled) RidePlanningStorage.tripDetails?.destination == null else false
```

Comment:
on iOS we set shouldReverseGeolocate as true if the whole trip info == null, not just the destination

Reply:
Fixed